src/loading/binary-loader.ts

```diff
@@ -46,7 +47,10 @@ export class BinaryLoader {
   xhrRequest: XhrRequest;
   callbacks: Callback[];
 
-  private workers: Worker[] = [];
+  private static readonly WORKER_POOL = new WorkerPool(
+    32,
```

Comment:
Is there a possibility to make this dynamic somehow? Such that the worker limit is configured via a config param on the loader? Because otherwise I think we might not be able to use the full capability of a multicore system.

Reply:
This is now addressed in my last commit.